src/components/Input/Input.tsx

```diff
@@ -0,0 +1,17 @@
+import "./input.css"
+import { HTMLInputTypeAttribute, InputHTMLAttributes } from "react"
+
+interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
+    title: string;
+}
+
+export default function Input({title, ...rest}: InputProps) {
+    return(
+        <>
+            <div className="container-input-component">
+                <p className="p-input-component">{title}</p>
+                <input className="input-component" {...rest}/>
+            </div>
+        </>
+    )
```

Comment:
Aqui o fragmento não é necessário.
```suggestion
    return(
        <div className="container-input-component">
            <p className="p-input-component">{title}</p>
            <input className="input-component" {...rest}/>
        </div>
    )
```